tools/shared/src/main/scala/org/scalajs/core/tools/linker/backend/emitter/Emitter.scala

```diff
@@ -202,7 +202,8 @@ final class Emitter private (semantics: Semantics, outputMode: OutputMode,
           implicit val pos = pos0
           val from = js.StringLiteral(module)
           val moduleBinding = jsGen.envModuleField(module).ident
-          val importStat = js.ImportNamespace(moduleBinding, from)
+          val bindings = List((js.ExportName(moduleBinding.name), moduleBinding))
+          val importStat = js.Import(bindings, from)
```

Comment:
The trees created here represent the following import:
```js
import { $i_someModule as $i_someModule } from "someModule"
```
I don't think this is going to work.

Reply:
I thought `moduleBinding.name` is `Button` in `import { Button } from "semantic-ui-react"`
but it is actualy an alias ?

Reply:
`moduleBinding.name` is always an encoded form of the module path (the part after `from`), as you can see in the computation of `jsGen.envModuleField`, and in how `module` is computed in `foreachImportedModule`.